modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js

```diff
@@ -911,9 +911,9 @@
 				pTabsId = isMinerva ? 'page-actions' :
 					$( '#p-views' ).length ? 'p-views' : 'p-cactions',
 				// Minerva puts the '#ca-...' ids on <a> nodes
-				$caSource = $( 'li#ca-viewsource' ),
-				$caEdit = $( 'li#ca-edit, li#page-actions-edit' ),
-				$caVeEdit = $( 'li#ca-ve-edit' ),
+				$caSource = $( '#ca-viewsource' ),
+				$caEdit = $( '#ca-edit, #page-actions-edit' ),
+				$caVeEdit = $( '#ca-ve-edit' ),
```

Comment:
Why the selector changes?

Reply:
In our current setup, this code for multi tab is executed so we need to match selectors for our skin. We could change config to make VE know we're not have multi tabs like on standard mediawiki skin but it introduces problems e.g. with section edits as we're have everything prepared for current config.

Reply:
If it ultimately makes things simpler, I think we should do it. If it's too large an effort to be done within the scope of this ticket, we could create a follow-up task for it.